packages/casting/src/follower-cosmjs.js

```diff
@@ -217,14 +221,13 @@ export const makeCosmjsFollower = (
   };
 
   /**
+   * @deprecated no longer supported https://github.com/cosmos/cosmjs/pull/1623
    * @param {number} [height]
    * @returns {Promise<QueryStoreResponse>}
    */
   const getProvenDataAtHeight = async height => {
-    return retryGetPrefixedData(async (endpoint, storeName, storeSubkey) => {
-      const queryClient = await provideQueryClient(endpoint);
-      return E(queryClient).queryStoreVerified(storeName, storeSubkey, height);
-    });
+    console.error('getProvenDataAtHeight', height, 'is no longer supported');
+    throw makeError(X`Verified queries are no longer supported`);
```

Comment:
This doesn't break anything? I'd expect at least that the old default "optimistic" starts showing a lot of errors/warnings, but I'm not sure where, and tests won't exactly catch noisy logs. Maybe we could make a temporary change to fail earlier if the proof is "optimistic" so we can see if any tests fail? Then we'd know which code is impacted.

Reply:
That would be a problem but I'm pretty sure this PR changes the default to 'none'.